_data/categories.yml

```diff
@@ -51,3 +51,9 @@
 
 - name: Conferences
   slug: conferences
+
+- name: Graduates
+  slug: graduates
+
+- name: Data Engineering
+  slug: data-engineering
```

Comment:
there's already a data category, and both of those categories need to be created in the /category directory